takecare19/takecare19
# Loading error states

I added skeleton loaders from resource cards (3) and the categories list. 
For all errors, i have it set to redirect to an error page. Not sure how to test this on the deployed version (I tested this by bunging up the database url in the code), but it looks like this:

![image](https://user-images.githubusercontent.com/30587540/80325771-83297380-8804-11ea-8f87-af6d7b100298.png)


###Testing instructions for loaders:
* On page load, confirm you can see the categories loader and resource card loaders before the real content loads
* When you click "load more" confirm that you see the resource card loaders where the new resources will be loaded in
